lib/ansible/modules/cloud/cloudscale/cloudscale_server.py

```diff
@@ -0,0 +1,438 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2017, Gaudenz Steinlin <gaudenz.steinlin@cloudscale.ch>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: cloudscale_server
+short_description: Manages servers on the cloudscale.ch IaaS service
+description:
+  - Create, start, stop and delete servers on the cloudscale.ch IaaS service
+  - All operations are performed using the cloudscale.ch public API v1
+  - "For details consult the full API documentation: U(https://www.cloudscale.ch/en/api/v1)"
+  - An valid API token is required for all operations. You can create as many tokens as you like using the cloudscale.ch control panel at U(https://control.cloudscale.ch)
+notes:
+  - Instead of the api_token parameter the CLOUDSCALE_API_TOKEN environment variable can be used.
+  - To create a new server at least the C(name), C(ssh_key), C(image) and C(flavor) options are required.
```

Comment:
See http://docs.ansible.com/ansible/dev_guide/developing_modules.html#documenting-your-module it means constant-width and is recommended for parameters in the documentation.